pages/howto/newenv.md

```diff
@@ -49,47 +49,41 @@ $ echo "module-version ccrsoft/2023.01 default" >> $HOME/.modulerc
 
-## How to use new compute resources?
+## What happened to the ubhpc-future reservation?  
+
+Since the default software environment has been switched to ccrsoft/2023.01, it is no longer necessary to keep the new compute nodes separate from the rest of the cluster.  The ubhpc-future reservation has been deleted and users can submit to the cluster without it.  Those using the latest software release can run on any node and those using the legacy software will need to [specify the `LEGACY` tag](#how-to-use-the-legacy-software) or their jobs will fail.  If you're using GPUs, please [see here](#how-to-properly-request-gpu-nodes) for guidance on how to request them properly.
+
+## How to properly request GPU nodes?
 
 CCR recently deployed brand new compute resources with new processors, memory,
-and GPUs. These nodes have been temporarily put into a reservation for users to
-access them and starting after the January maintenance downtime you will
-no longer need to specify the reservation. 
+and GPUs.   The new GPU nodes have the latest NVIDIA drivers installed on them and should be used with the latest software release. To ensure your job gets allocated on a newer GPU node, we recommend specifying `--constraint=NONLEGACY` in your Slurm job script or in your interactive session request.  If using OnDemand, you can specify `NONLEGACY` in the "Node Features" box of the application forms.
 
-Currently, you will need to specify the following in your batch scripts: 
+ For more info on requesting GPUs [see here](../hpc/jobs.md#slurm-directives-partitions--qos):  
 
-```
-#SBATCH --cluster=ub-hpc
-#SBATCH --partition=general-compute
-#SBATCH --qos=general-compute
-#SBATCH --reservation=ubhpc-future
 
-module load ccrsoft/2023.01
-```
+!!! Tip "Use the latest software environment for GPU use"  
+    Legacy software can not be used with the new nodes due to driver incompatibilities.  If you must use legacy software, use the `LEGACY` constraint when requesting a GPU.     
 
-!!! Tip "Use the latest ccrsoft/2023.01"  
-    These new nodes work best if using the latest ccrsoft release. 
 
 ## New login node vortex-future is... the future?
 
 Users now have access to a new login node: `vortex-future.ccr.buffalo.edu`.
 This login node will always contain the latest and greatest features CCR has to
-offer and serves as a preview of the next deployment of all login nodes. 
+offer and serves as a preview of the next deployment of all login nodes.  You may continue to use `vortex.ccr.buffalo.edu` which loads the default ccrsoft\2023.01 software release as well.  
```

Comment:
Minor typo. Change backslash to forward slash:  `ccrsoft/2023.01`

Reply:
Fixed!